OneSignalSDK/onesignal/core/src/main/java/com/onesignal/user/internal/operations/impl/executors/LoginUserFromSubscriptionOperationExecutor.kt

```diff
@@ -82,8 +84,10 @@ internal class LoginUserFromSubscriptionOperationExecutor(
             return when (responseType) {
                 NetworkUtils.ResponseStatusType.RETRYABLE ->
                     ExecutionResponse(ExecutionResult.FAIL_RETRY)
-                NetworkUtils.ResponseStatusType.UNAUTHORIZED ->
+                NetworkUtils.ResponseStatusType.UNAUTHORIZED -> {
+                    _identityModelStore.invalidateJwt()
```

Comment:
I see we call this `_identityModelStore.invalidateJwt()` in every executor, do you think it would work out to have the OperationRepo process `ExecutionResult.FAIL_UNAUTHORIZED` so it is the only one calling `invalidateJwt()`?

Reply:
Addressed in the fixup commit. The original plan was to fire the invalidate event for some specific operations instead of any operation. but I just realized UNAUTHORIZED response can only be returned by these specific operations anyway. I just move the invalidation to OperationRepo out of executors.